### DISCUSSION TITLE
Stop restarting Embedding on Configuration Change

### DISCUSSION THREAD

**Discussion by hittjwX** [brianpetro/obsidian-smart-connections#757] [Q&A]
Version v2.1.99 is not working for me. I'm in configuration looking at settings. If I change something it immediately starts `Importing...`. Can you make the embed processing wait till I at least close the configuration panel. Damn thing keeps restarting to apply settings.

**Comment by brianpetro** [comment]
@jwhco I know it's an undesirable UX at the moment. The underlying components are in the process of being improved, and this will prevent the necessary restarts in the future 🌴
